interface/usergroup/adminacl.php

```diff
@@ -28,6 +29,7 @@
         
 <html>
 <head>
+  <?php call_required_libraries(array("bootstrap"))?>
  <script type="text/JavaScript" src="../../library/js/jquery121.js"></script>
```

Comment:
Use the above call to pull the jquery from /assets.  If you do one, do them all.